src/_includes/footer.html

```diff
@@ -1,18 +1,46 @@
-<hr>
-  <div class="row">
-      <div class="col-xs-12">
-          <footer>
-              <p class="text-center">
-                &copy; Copyright
-                <a href="http://www.apache.org">The Apache Software Foundation</a>,
-                {{ 'now' | date: "%Y" }}. All Rights Reserved.
-              </p>
-              <p class="text-center">
-                <a href="{{ site.baseurl }}/privacy_policy">Privacy Policy</a> |
-                <a href="{{ "/feed.xml" | prepend: site.baseurl }}">RSS Feed</a>
-              </p>
-          </footer>
+<footer class="footer">
+  <div class="footer__contained">
+    <div class="footer__cols">
+      <div class="footer__cols__col">
+        <div class="footer__cols__col__logo">
+          <img src="/images/beam_logo_circle.svg" class="footer__logo">
+        </div>
+        <div class="footer__cols__col__logo">
+          <img src="/images/apache_logo_circle.svg" class="footer__logo">
+        </div>
       </div>
+      <div class="footer__cols__col footer__cols__col--md">
+        <div class="footer__cols__col__title">Start</div>
+        <div class="footer__cols__col__link"><a href="#TODO">Overview</a></div>
+        <div class="footer__cols__col__link"><a href="#TODO">Quickstart</a></div>
+        <div class="footer__cols__col__link"><a href="#TODO">Walkthroughs</a></div>
+        <div class="footer__cols__col__link"><a href="#TODO">Downloads</a></div>
+      </div>
+      <div class="footer__cols__col footer__cols__col--md">
+        <div class="footer__cols__col__title">Docs</div>
+        <div class="footer__cols__col__link"><a href="#TODO">Concepts</a></div>
+        <div class="footer__cols__col__link"><a href="#TODO">Pipelines</a></div>
+        <div class="footer__cols__col__link"><a href="#TODO">Runners</a></div>
+        <div class="footer__cols__col__link"><a href="#TODO">API Reference</a></div>
+      </div>
+      <div class="footer__cols__col footer__cols__col--md">
+        <div class="footer__cols__col__title">Community</div>
+        <div class="footer__cols__col__link"><a href="#TODO">Contribute</a></div>
+        <div class="footer__cols__col__link"><a href="#TODO">Team</a></div>
+        <div class="footer__cols__col__link"><a href="#TODO">Media</a></div>
+      </div>
+      <div class="footer__cols__col footer__cols__col--md">
+        <div class="footer__cols__col__title">Resources</div>
+        <div class="footer__cols__col__link"><a href="#TODO">Blog</a></div>
+        <div class="footer__cols__col__link"><a href="#TODO">Support</a></div>
+        <div class="footer__cols__col__link"><a href="https://github.com/apache/beam">GitHub</a></div>
+      </div>
+    </div>
+  </div>
+  <div class="footer__bottom">
+    &copy;
+    <a href="http://www.apache.org">The Apache Software Foundation</a>
+    | <a href="{{ site.baseurl }}/privacy_policy">Privacy Policy</a>
+    | <a href="{{ "/feed.xml" | prepend: site.baseurl }}">RSS Feed</a>
```

Comment:
Mention trademarks: Apache Flink, Apache, and the Apache feather logo are either registered trademarks or trademarks of The Apache Software Foundation.